README.md

```diff
@@ -26,6 +26,14 @@ Just execute:
 
     $ ./build.js
 
+In the occassion you are running a windows platform, we will have to make a few fixes for your machine to be able to build this app:
+    1. Please make sure you have installed git on your machine.
+    2. Open cmd with administrator privilages (Right click on cmd -> run as administrator).
+    3. Set your current working directory to this folder ( "cd {path root of polypod}/windows") and run the convert-symlinks.js file with node. 
```

Comment:
You are telling it they should have "git", and then to run something that presupposes they a) know how to to do that and b) assumes node is installed...

Reply:
node is mentioned in the requirements further up so I guess since this is a specific case of using windows then it includes the original requirements too :D. 
In terms of how they should install git, should I not leave it to them to find out? I could include and set it in your environment variable path, and then I guess a quick google search will quickly point them in the right direction.

Reply:
I would rather just eliminate the requirement for git. First, they don't really need it, they could have just downloaded it as a tar file. Second, if they are here they probably have it installed and know how to use it. What was a bit jarring to me was to assume that they don't know that they need git, and then 2 steps below you assume they know how to run a JS script. You're probably safe assuming they know that, but then it's never a bad idea to include the exact command line they need to use (invoking node explicitly, for instance)

Reply:
I see what you mean but the reason I explicitly ask for them to have git is because of the `git ls` command that is inside the script, but I do think it is safe to assume they have it :)